frdrpc/go.mod

```diff
@@ -0,0 +1,17 @@
+module github.com/lightninglabs/faraday/frdrpc
+
+go 1.22.3
```

Comment:
nit: when we're touching or creating all these `go.mod` files, can we please also sync the place we put the version? Either all at the top or the bottom of the file? Instinctively I'd say probably at the bottom...